index.js

```diff
@@ -64,27 +67,34 @@ function checkGuess() {
 
 function hideAllMessages() {
   for (let elementIndex = 0; elementIndex <= messages.length; elementIndex++) {
-    messages[elementIndex].style.display = 'none';
+    messages[elementIndex] = 'none';
   }
 }
 
-funtion setup() {
+function setup() {
   // Get random number
   targetNumber = getRandomNumber(1, 100);
   console.log(`target number: ${targetNumber}`);
 
-  // Reset number of attempts
-  maxNumberOfAttempts = 0;
-
+  // Reset number of attempts for guessing the right number
+  attempts = 0;
+  
   // Enable the input and submit button
-  submitButton.disabeld = false;
+  submitButton.disabled = false;
   guessInput.disabled = false;
-
-  hideAllMessages();
```

Comment:
We should add this `hideAllMessages();` call back once the fix to the function is done (see above comment). Then we can get rid of lines 87-92 which are manually setting the messages to 'none'.